pom.xml

```diff
@@ -664,7 +671,6 @@ THE SOFTWARE.
                 <bannedDependencies>
                   <excludes>
                     <exclude>org.sonatype.sisu:sisu-guice</exclude>
-                    <exclude>log4j:log4j</exclude>
```

Comment:
OK, hopefully plugins which override the scope on purpose (and changed it to compile or runtime, test does not matter) might ignore the warning as well, but a warning might be a good idea nontheless. Then I would include `commons-logging:commons-logging` as well?